From d6fe1cf89026d3111112ee29ef8d93cd7a8aaa24 Mon Sep 17 00:00:00 2001
Message-Id: <d6fe1cf89026d3111112ee29ef8d93cd7a8aaa24.1648734782.git.geliang.tang@suse.com>
From: Xu Kuohai <xukuohai@huawei.com>
Date: Thu, 25 Mar 2021 13:41:41 +0000
Subject: [PATCH] bpf: Fix a spelling typo in bpf_atomic_alu_string disasm
Git-commit: d6fe1cf89026d3111112ee29ef8d93cd7a8aaa24
Patch-mainline: v5.12-rc7
References: bsn#127
Modified-by-SEL: No


The name string for BPF_XOR is "xor", not "or". Fix it.

Fixes: 981f94c3e921 ("bpf: Add bitwise atomic instructions")
Signed-off-by: Xu Kuohai <xukuohai@huawei.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Acked-by: Brendan Jackman <jackmanb@google.com>
Link: https://lore.kernel.org/bpf/20210325134141.8533-1-xukuohai@huawei.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 kernel/bpf/disasm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/bpf/disasm.c b/kernel/bpf/disasm.c
index 3acc7e0b6916..faa54d58972c 100644
--- a/kernel/bpf/disasm.c
+++ b/kernel/bpf/disasm.c
@@ -84,7 +84,7 @@ static const char *const bpf_atomic_alu_string[16] = {
 	[BPF_ADD >> 4]  = "add",
 	[BPF_AND >> 4]  = "and",
 	[BPF_OR >> 4]  = "or",
-	[BPF_XOR >> 4]  = "or",
+	[BPF_XOR >> 4]  = "xor",
 };
 
 static const char *const bpf_ldst_string[] = {
-- 
2.34.1

